packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -1203,4 +1203,102 @@ void main() {
       await testColor(color);
     }
   });
+  testWidgets('RefreshIndicator default elevation value correctly',
+    (WidgetTester tester) async {
+  const List<double> positions = <double>[50.0, 100.0, 150.0];
+
+    final AnimationController positionController =
+        AnimationController(vsync: const TestVSync());
+    addTearDown(positionController.dispose);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: RefreshIndicator(
+          onRefresh: refresh,
+          child: ListView(
+            physics: const AlwaysScrollableScrollPhysics(),
+            children: const <Widget>[Text('X')],
+          ),
+        ),
+      ),
+    );
+
+    RefreshProgressIndicator getIndicator() {
+      return tester.widget<RefreshProgressIndicator>(
+        find.byType(RefreshProgressIndicator),
+      );
+    }
+
+    // Correspond to [_kDragContainerExtentPercentage].
```

Comment:
```suggestion
    // Corresponds to [_kDragContainerExtentPercentage].
```

Reply:
This shouldn't be resolved, it still says "Correspond". Also you can probably remove the square brackets since they don't do anything in regular comments, only `///` doc comments.